haskell/src/Indigo/Contracts/Transferlist/External.hs

```diff
@@ -0,0 +1,79 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+-- | This a dummy file that resembles lorentz-contract-transferlist parameter needed for origination.
+--
+-- For more information you can refer to
+-- https://github.com/tqtezos/lorentz-contract-transferlist/tree/476c0429adf74041ed7d26c44e5e8e73f5a54f36
+
+module Indigo.Contracts.Transferlist.External
+  ( Storage (..)
+  , convertToExternalStorage
+  ) where
+
+import qualified Data.List as List
+import qualified Data.Map as Map
+import qualified Data.Set as Set
+
+import qualified Indigo.Contracts.Transferlist.Internal as Internal
+import Lorentz as L
+
+type TransferlistId = Natural
+
+data OutboundTransferlists = OutboundTransferlists
+  { unrestricted :: Bool
+  , allowedTransferlists :: Set TransferlistId
+  }
+  deriving stock (Generic, Show)
+  deriving anyclass (IsoValue, HasTypeAnn)
+
+data Storage = Storage
+  { issuer :: Address
+  , users  :: BigMap Address TransferlistId
+  , transferlists :: BigMap TransferlistId OutboundTransferlists
+  , admin :: Address
+  }
+  deriving stock Generic
+  deriving anyclass (IsoValue, HasTypeAnn)
+
+{-# ANN convertToExternalStorage ("HLint: ignore Use 'head' from Universum.Unsafe" :: Text) #-}
```

Comment:
Why don't you want to use `Unsafe.head` here btw?

Reply:
This leads adding `universum` to dependencies

Reply:
I mean `Unsafe` from `morley-prelude`, not `Universum.Unsafe` from `universum`: https://hackage.haskell.org/package/morley-prelude-0.3.0/docs/Unsafe.html

That being said, maybe `hlint` will trigger this warning for `Unsafe` because it's not exactly `Universum.Unsafe`, but I'd try at least.